apps/web/components/email-list/EmailList.tsx

```diff
@@ -472,6 +492,25 @@ export function EmailList(props: {
                   />
                 );
               })}
+              {/* Load more button to fetch more data */}
+              {isLoadMore && (
+                <div className="pb-2">
+                  <Button
+                    variant="outline"
+                    className="w-full py-4"
+                    onClick={handleLoadMoreThreads}
+                  >
+                    {isLoadMoreLoading ? (
+                      <ButtonLoader />
+                    ) : (
+                      <>
+                        {" "}
```

Comment:
Why the space here?
From when I ran it the UI didn't look great here.

To be more inline with the rest of the UI, you could have a spinner replace the icon. And have the icon to the left.

See the `show more` button on the Bulk Unsubscribe page for example:
![CleanShot 2024-08-29 at 09 32 30@2x](https://github.com/user-attachments/assets/18818365-2fb5-47c7-8a94-2cdb073f0c50)

Reply:
Done!

The <ButtonLoader/> component has some css issue, it wobbles at it's axis. Saw it at a few other places in the UI. 
One Simple fix was to change the size to 5, but then it would have caused UI inconsistency so I didn't touch it!
Just letting you know!